cyvcf2/cyvcf2.pyx

```diff
@@ -130,7 +133,87 @@ cdef set_constants(VCF v):
         v.HOM_ALT = 3
 
 
-cdef class VCF:
+
+cdef class HTSFile:
+
+    cdef htsFile *hts
+    cdef bytes fname
+    cdef bytes mode
+    cdef bint from_path
```

Comment:
there's probably more in `VCF` that's needed across both `VCF` and `Writer` that we can move here.